Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py

```diff
@@ -299,9 +301,12 @@ def invoke(args, aws_client):
     if 'FunctionError' in response:
         data.update({'FunctionError': response['FunctionError']})
 
-    ec = {'AWS.Lambda.InvokedFunctions(val.FunctionName === obj.FunctionName)': data}
     human_readable = tableToMarkdown('AWS Lambda Invoked Functions', data)
-    return_outputs(human_readable, ec)
+    return CommandResults(
+        outputs=data,
+        readable_output=human_readable,
+        outputs_prefix="AWS.Lambda.InvokedFunctions"
```

Comment:
```suggestion
        outputs_prefix="AWS.Lambda.InvokedFunctions",
        outputs_key_field=["FunctionName", "Payload"]
```
I understand the reason you want it to be added every run, on the other hand, if you run the same FunctionName and the same Payload I think you would want it to replace the existing one,
That's why I propose to add that only if both are equal then it will be replaced and if one of them is different then a new one will be added.
What do you think?